authorization/role/repository/role_mapping_blackbox_test.go

```diff
@@ -58,6 +58,48 @@ func (s *roleMappingBlackBoxTest) TestOKToDelete() {
 	}
 }
 
+func (s *roleMappingBlackBoxTest) TestOKToDeleteForResource() {
+	g := s.NewTestGraph()
+
+	// Create 5 role mappings
+	r := g.CreateResource(g.ID("r"))
+	for i := 0; i < 5; i++ {
+		g.CreateRoleMapping(r)
+	}
+
+	// Check all of them are present
+	mappings, err := s.repo.FindForResource(s.Ctx, g.ResourceByID("r").Resource().ResourceID)
```

Comment:
The "r" resource has already been assigned to a variable `r`, why not use that instead?

`mappings, err := s.repo.FindForResource(s.Ctx, r.Resource().ResourceID)`

Reply:
Right. Done in https://github.com/fabric8-services/fabric8-auth/pull/498/commits/a82bf3fd432bb4f3f5f7307f60d073c1a1cc3716